map_view/display-device-location/src/main/java/com/esri/samples/display_device_location/DisplayDeviceLocationLauncher.java

```diff
@@ -0,0 +1,11 @@
+package com.esri.samples.display_device_location;
```

Comment:
Alan suggested adding a copyright statement to these files in a previous review. Something to consider @Rachael-E @TADraeseke ?